src/plugins/intel_cpu/tests/functional/test_utils/arm/cpu_test_utils.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils/cpu_test_utils.hpp"
+#include "ie_ngraph_utils.hpp"
+#include "openvino/core/type/element_type.hpp"
+#include "utils/rt_info/memory_formats_attribute.hpp"
+#include "utils/general_utils.h"
+#include <cstdint>
+
+namespace CPUTestUtils {
+
+std::vector<CPUSpecificParams> filterCPUInfo(const std::vector<CPUSpecificParams>& CPUParams) {
+    std::vector<CPUSpecificParams> archCPUParams = filterCPUInfoForArch(CPUParams);
+    std::vector<CPUSpecificParams> deviceCPUParams = filterCPUInfoForDevice(archCPUParams);
+    return deviceCPUParams;
+}
+
+std::vector<CPUSpecificParams> filterCPUInfoForArch(const std::vector<CPUSpecificParams>& CPUParams) {
+    std::vector<CPUSpecificParams> resCPUParams;
+    const int selectedTypeIndex = 3;
+
+    for (auto param : CPUParams) {
+        auto selectedTypeStr = std::get<selectedTypeIndex>(param);
+
+        if (selectedTypeStr.find("jit") != std::string::npos)
```

Comment:
I think the arm part should not care / know about x86, so it should filter out everything non-arm related (non-acl for a current state)
So if not acl (maybe also ref?) -> than remove.